.travis.yml

```diff
@@ -9,7 +9,7 @@ before_install:
   - sudo dpkg --add-architecture i386
   - sudo apt-get update
   - sudo apt-get install --no-install-recommends -y wine
-  - pip install pytest pytest-cov codecov mock nbval matplotlib
+  - pip install pytest pytest-cov codecov mock nbval matplotlib git+https://github.com/thedataincubator/expectexception.git@master
```

Comment:
`expectexception` means that we can put cells which raise errors in our example notebooks but still have them under sensible CI